_posts/news/2020-10-16-announcing-scalajs-1.3.0.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+title: Announcing Scala.js 1.3.0
+category: news
+tags: [releases]
+permalink: /news/2020/10/16/announcing-scalajs-1.3.0/
+---
+
+
+We are excited to announce the release of Scala.js 1.3.0!
+
+This release brings one of the most awaited features for Scala.js: module splitting support!
+It is now possible to split the generated .js file in multiple modules, to optimize download size in multi-page applications or speed up incremental bundling.
+
+In addition, this release contains a number of new methods and classes in the JDK implementation, among which the higher-order methods of the `java.util` collections, and the locale-sensitive overloads of `String.toLowerCase`, `toUpperCase` and `format`.
+
+Read on for more details.
+
+<!--more-->
+
+## Getting started
+
+If you are new to Scala.js, head over to [the tutorial]({{ BASE_PATH }}/tutorial/).
+
+If you need help with anything related to Scala.js, you may find our community [on Gitter](https://gitter.im/scala-js/scala-js) and [on Stack Overflow](https://stackoverflow.com/questions/tagged/scala.js).
+
+Bug reports can be filed [on GitHub](https://github.com/scala-js/scala-js/issues).
+
+## Release notes
+
+If upgrading from Scala.js 0.6.x, make sure to read [the release notes of Scala.js 1.0.0]({{ BASE_PATH }}/news/2020/02/25/announcing-scalajs-1.0.0/) first, as they contain a host of important information, including breaking changes.
+
+This is a **minor** release:
+
+* It is backward binary compatible with all earlier versions in the 1.x series: libraries compiled with 1.0.x through 1.2.x can be used with 1.3.0 without change.
+* It is *not* forward binary compatible with 1.2.x: libraries compiled with 1.3.0 cannot be used with 1.2.x or earlier.
+* It is *not* entirely backward source compatible: it is not guaranteed that a codebase will compile *as is* when upgrading from 1.2.x (in particular in the presence of `-Xfatal-warnings`).
+
+As a reminder, libraries compiled with 0.6.x cannot be used with Scala.js 1.x; they must be republished with 1.x first.
+
+## Module splitting
+
+### Quickstart
+
+Instead of `fastOptJS` / `fullOptJS`, use `fastLinkJS` / `fullLinkJS` and do any or all of the following:
+
+#### For many small modules
+
+{% highlight scala %}
+import org.scalajs.linker.interface.ModuleSplitStyle
+scalaJSLinkerConfig ~= (_.withModuleSplitStyle(ModuleSplitStyle.SmallestModules))
+{% endhighlight %}
+
+#### For different entry points
```

Comment:
IMO this should come first. That use case is entirely usefully satisfied by Scala.js core. The "many small modules" use case is useful only when piped through a bundler down the line.